### ISSUE TITLE
[Bug]: No autocompletion for validation (Form Request

### ISSUE THREAD

**Issue by SirMishaa** [laravel-idea/plugin#544]
### Bug description

Hello, 

Like you can I see, I have no autocompletion for my class ``GetQuoteRequest`` :(
I'm a just on a simple Laravel project https://github.com/SirMishaa/yago-challenge

Am I doing something wrong? 
![image](https://user-images.githubusercontent.com/42717232/166877055-821305e0-a621-41a7-b1a4-54abdbb1185f.png)

### Plugin version

5.3.1.221

### Operating system

Linux

**Comment by SirMishaa** [comment]
Nvm, it works now... I just needed to write a bit... (like the first two rules of validation, and now I have autocomplete... weird 
![image](https://user-images.githubusercontent.com/42717232/166877470-34ec8c0d-0fa8-4b7b-a721-6b85713852f1.png)

**Comment by adelf** [comment]
That's strange. Actually, I have some reports about the same behavior - "floating" completion for validation rules in FormRequests... but I still can't reproduce this bug.

Could you share your PhpStorm logs? main menu - Help - Show log in Explorer?

**Comment by SirMishaa** [comment]
[idea.log](https://github.com/laravel-idea/plugin/files/8629542/idea.log)

Here you go
And yes, sometime you have autocompletion, sometime not, it's a bit weird.

**Comment by Allgoodnamesargone** [comment]
Now this is happening on my side ALL the time

**Comment by hexlay** [comment]
I have the same problem

**Comment by KKJoshLo** [comment]
Same on me, and I've never seen the autocompletion of validation.

**Comment by adelf** [comment]
I Couldn't reproduce the issue, but I hope I found the reason. Will try to fix it soon.

**Comment by adelf** [comment]
Please try the new 6.2.3 version.

**Comment by Allgoodnamesargone** [comment]
> Please try the new 6.2.3 version.

It's not fixed.

**Comment by GringoDotDev** [comment]
Same here. Version 6.2.4.223.

To be precise - it *does* autocomplete the rules (min, max, numeric, etc.). It *does not* autocomplete the field names (firstName, etc.).

**Comment by entity** [comment]
> Same here. Version 6.2.4.223.
> 
> To be precise - it _does_ autocomplete the rules (min, max, numeric, etc.). It _does not_ autocomplete the field names (firstName, etc.).

Just faced this, i'm unable to auto-complete the form request fields in my controller. Will this ever be a feature? @adelf

**Comment by adelf** [comment]
@heychazza you mean "$request->firstName" completion?

**Comment by entity** [comment]
> @heychazza you mean "$request->firstName" completion?

Yep exactly, if you’ve got a form request, then in your controller try to access a field

**Comment by gorbunov** [comment]
Fixed some time ago!
